services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/pdf/HaittojenhallintasuunnitelmaPdfEncoder.kt

```diff
@@ -64,19 +71,193 @@ class HaittojenhallintasuunnitelmaPdfEncoder(private val mapGenerator: MapGenera
             row("Työn alkupäivämäärä", data.startTime.format())
             row("Työn loppupäivämäärä", data.endTime.format())
         }
+
+        hankealueet.forEach { hankealue ->
+            val kaivuilmoitusalue = data.areas.first { it.hankealueId == hankealue.id }
+            val worstIndexes = kaivuilmoitusalue.worstCasesInTormaystarkastelut()
+
+            document.section(hankealue.nimi) {
+                row("Toimet työalueiden haittojen hallintaan, ${hankealue.nimi}", toimetTitle())
+                row("", nuisanceScore(worstIndexes?.liikennehaittaindeksi?.indeksi))
+                val toimet =
+                    kaivuilmoitusalue.haittojenhallintasuunnitelma[Haittojenhallintatyyppi.YLEINEN]
+                row("", toimet(toimet))
+                row("", "")
+            }
+
+            document.section(null) {
+                row(
+                    "Linja-autojen paikallisliikenne, ${hankealue.nimi}",
+                    map(kaivuilmoitusalue, hankealue) { it?.linjaautoliikenneindeksi },
+                )
+                row("", toimetTitle())
+                row("", nuisanceScore(worstIndexes?.linjaautoliikenneindeksi))
+                val toimet =
+                    kaivuilmoitusalue.haittojenhallintasuunnitelma[
+                            Haittojenhallintatyyppi.LINJAAUTOLIIKENNE]
+                row("", toimet(toimet))
+                row("", "")
+            }
+
+            document.section(null) {
+                row(
+                    "Autoliikenteen ruuhkautuminen, ${hankealue.nimi}",
+                    map(kaivuilmoitusalue, hankealue) { it?.autoliikenne?.indeksi },
+                )
+                row("", toimetTitle())
+                row("", autoliikennehaitat(worstIndexes?.autoliikenne))
+                val toimet =
+                    kaivuilmoitusalue.haittojenhallintasuunnitelma[
+                            Haittojenhallintatyyppi.AUTOLIIKENNE]
+                row("", toimet(toimet))
+                row("", "")
+            }
+
+            document.section(null) {
+                row(
+                    "Pyöräliikenteen merkittävyys, ${hankealue.nimi}",
+                    map(kaivuilmoitusalue, hankealue) { it?.pyoraliikenneindeksi },
+                )
+                row("", toimetTitle())
+                row("", nuisanceScore(worstIndexes?.pyoraliikenneindeksi))
+                val toimet =
+                    kaivuilmoitusalue.haittojenhallintasuunnitelma[
+                            Haittojenhallintatyyppi.PYORALIIKENNE]
+                row("", toimet(toimet))
+                row("", "")
+            }
+
+            document.section(null) {
+                row(
+                    "Raitiovaunuliikenne, ${hankealue.nimi}",
```

Comment:
There was a discussion about the different terms "Raitioliikenne" (used in hanke haittojenhallinta) and "Raitiovaunuliikenne" (used in kaivuilmoitus haittojenhallinta). The first one should be used. See https://helsinkicity.slack.com/archives/C01AYUN0XD5/p1734095085946859

Reply:
I changed these to Raitioliikenne. Thank you for noticing.